Jenkinsfile

```diff
@@ -0,0 +1,18 @@
+pipeline {
+	agent { node { label 'ansible' } }
+	stages {
+		stage('ansible-lint') {
+			steps {
+				ansiColor('xterm') {
+					sh '''#!/bin/bash -xe
+					find -name '*.yml' >scripts
```

Comment:
This could also be `scripts=$(find . -name '*.yml')`, then we don't need the temporary file